src/PIL/Image.py

```diff
@@ -847,7 +881,7 @@ def load(self):
         operations. See :ref:`file-handling` for more information.
 
         :returns: An image access object.
-        :rtype: :ref:`PixelAccess` or :py:class:`PIL.PyAccess`
+        :rtype: :py:class:`.PixelAccess` or :py:class:`.PyAccess`
```

Comment:
My initial reaction to this was to feel confused , because the return type is PixelAccess or None, but this says PixelAccess or PyAccess.

Of course, the return type uses PixelAccess the protocol, not the class. But should the protocol have a different name perhaps to avoid confusion? `SupportsPixelAccess`?

Reply:
Be aware that this will become simpler after Pillow 10.4.0, as PyAccess will end deprecation and be removed.

Reply:
I think that after 10.4.0, we'll have no need for the protocol - we can just use `core.PixelAccess`. If so, then it's not helpful to add a public protocol only to remove it in the next version. I've created https://github.com/nulano/Pillow/pull/39 to return `core.PixelAccess | PyAccess.PyAccess | None` instead.

Reply:
Fair enough, I've pushed a similar commit, see comment in https://github.com/nulano/Pillow/pull/39.